gluster/file/cmd/glusterfile-provisioner/glusterfile-provisioner.go

```diff
@@ -102,14 +102,14 @@ func (p *glusterfileProvisioner) GetAccessModes() []v1.PersistentVolumeAccessMod
 func (p *glusterfileProvisioner) Provision(options controller.VolumeOptions) (*v1.PersistentVolume, error) {
 
 	if options.PVC.Spec.Selector != nil {
-		return nil, fmt.Errorf("claim Selector is not supported")
+		return nil, fmt.Errorf("claim Selectors are not supported")
```

Comment:
afaict, only one claim selector can be specified.

Reply:
...oh right. One selector can include any number of labels. Duh. :)